pyproject.toml

```diff
@@ -355,9 +355,10 @@ module = [
     'napari.layers.shapes._shapes_mouse_bindings',
     'napari.layers.shapes._shapes_utils',
     'napari.layers.shapes.shapes',
-    'napari.layers.tracks._track_utils',
-    'napari.layers.tracks._tracks_key_bindings',
-    'napari.layers.tracks.tracks',
+    'napari.layers.surface._surface_key_bindings',
```

Comment:
Why do these need to be added in this PR? Because of the typing change to `colormap`? Could we avoid that in this PR? Or would merging https://github.com/napari/napari/pull/5832 resolve this?

Reply:
I've came across this problem as well in some of my PRs, I think that' because by default mypy will not propagate checks unless you pass the `--check-untyped-defs`, and thus adding types in some files often make other's fails. I think it's a fools errand to try to fix all types in the same PR for now, so I'm for moving forward even if it means temporarily re-ignoring some files.

Reply:
🙈 I think this was a rebase gone wrong and these lines shouldn't have been added, thanks for catching and I've removed them now 😄